<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сжатие речевых данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="decompressPanel" native="true">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Сжатие</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Длина сигнала:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>BPS:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Частота дискретизации:</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="inSignalLengthLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="inSignalBpsLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="inSignalRateLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="compressBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Сжать файл</string>
       </property>
      </widget>
      <widget class="QComboBox" name="pauseMatrixCombo">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="signalMatrixCombo">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="PEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="pEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>113</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Матрица для пауз</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>144</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Матрица для сиг.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>173</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>202</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>p</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>232</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>BPS</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bpsEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="compresPanel" native="true">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Распаковка</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Np</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ns</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>исх. BPS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>сж. BPS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="decompressBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Восстановить</string>
       </property>
      </widget>
      <widget class="QLabel" name="NpLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="RpLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="NsLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="RsLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="signalBpsLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="compressedBpsLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="loadWav"/>
    <addaction name="loadCwf"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="loadMatrixPath"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadWav">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Загрузить речь</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="loadCwf">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Загрузить сжатый файл</string>
   </property>
  </action>
  <action name="exit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="loadMatrixPath">
   <property name="text">
    <string>Установить директорию матриц</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
